packages/web/config/chain-infos.ts

```diff
@@ -2484,7 +2484,37 @@ const chainInfos = (
         },
       ],
       features: ["ibc-transfer", "ibc-go"],
-      explorerUrlToTx: "https://explorer.dys.dysonprotocol.com/dyson/tx/{txHash}",
+      explorerUrlToTx:
+        "https://explorer.dys.dysonprotocol.com/dyson/tx/{txHash}",
+    },
+    {
+      rpc: "https://testnet-rpc.marsprotocol.io/",
+      rest: "https://testnet-rest.marsprotocol.io/",
+      chainId: "ares-1",
+      chainName: "Mars Hub Testnet",
+      bip44: {
+        coinType: 118,
+      },
+      bech32Config: Bech32Address.defaultBech32Config("mars"),
+      currencies: [
+        {
+          coinDenom: "MARS",
+          coinMinimalDenom: "umars",
+          coinDecimals: 6,
+          coinGeckoId: "mars",
```

Comment:
It looks like this is referring to a different token on CoinGecko and thus should be removed.

Reply:
You are right, Mars Protocols coinGeckoId is `marsprotocol` but it's currently disabled, until mainnet launch